libs/novui/src/tokens/semanticColors.tokens.ts

```diff
@@ -372,4 +372,24 @@ export const COLOR_SEMANTIC_TOKENS = defineSemanticTokens.colors({
       type: 'color',
     },
   },
+  toggle: {
+    background: {
+      value: '{colors.legacy.B30}',
+      type: 'color',
+    },
+    thumb: {
+      value: '{colors.legacy.white}',
+      type: 'color',
+    },
+    checked: {
+      value: '{colors.green.30.dark}',
+      type: 'color',
+    },
+  },
+  sidebar: {
+    background: {
+      value: '{colors.legacy.B17}',
+      type: 'color',
+    },
+  },
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** We are no longer adding light mode at all? I wonder if we should still have the `base`... Not sure how it works behind the scenes on this ‏

Reply:
Good question @antonjoel82, how do we handle this?

Reply:
Please look at above examples with `_dark` defined

Reply:
Yeah I was more thinking of if we need to support it now as it seems like we will only be doing dark theme